scenes/menu/main_menu.gd

```diff
@@ -275,13 +269,16 @@ func _on_join_button_pressed():
 	var chosen_deck_id = chosen_deck['id']
 	if player_selected_character.begins_with("random"):
 		chosen_deck_id = player_selected_character + "#" + chosen_deck_id
-	NetworkManager.join_room(player_name, room_name, chosen_deck_id, 
-		GlobalSettings.CustomStartingTimer, GlobalSettings.EnforceTimer)
+	NetworkManager.join_room(player_name, 
+		room_name, 
+		chosen_deck_id, 
+		GlobalSettings.CustomStartingTimer, 
```

Comment:
This is wrong.
You need to be reading the settings out of "data", otherwise the 2 players won't have the same settings

Reply:
nevermind, wrong function!